notebooks/week5/02.train_model.py

```
@@ -0,0 +1,164 @@
+    run_id = run.info.run_id
+    mlflow.pyfunc.log_model(
+        python_model=wrapped_model,
+        artifact_path="pyfunc-power-consumption-model",
+        code_paths = [f"{root_path}/notebooks/power_consumption-0.0.1-py3-none-any.whl"],
+        signature=signature,
+        conda_env=conda_env
+    )
+
+
+model_uri=f'runs:/{run_id}/pyfunc-power-consumption-model'
+dbutils.jobs.taskValues.set(key="new_model_uri", value=model_uri)
```

Ruff

Undefined name `dbutils`

(F821)